__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should check TARGET_MIGRATION_VERSION to 5', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(6);
+  it('TEST should check TARGET_MIGRATION_VERSION to 7', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(7);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case has been updated to check if `TARGET_MIGRATION_VERSION` is 7, which aligns with the PR description stating that a new migration version 7 has been introduced. This change seems appropriate.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(6);
+    expect(TARGET_MIGRATION_VERSION).toBe(7);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->